Code/Editor/editor_lib_files.cmake

```diff
@@ -803,8 +803,8 @@ set(FILES
     EditorViewportCamera.h
     ViewportManipulatorController.cpp
     ViewportManipulatorController.h
-    LegacyViewportCameraController.cpp
-    LegacyViewportCameraController.h
+    RenderViewport.cpp
+    RenderViewport.h
```

Comment:
This file was deleted by https://github.com/o3de/o3de/pull/2840 I believe - bad merge?

Reply:
Yes, my bad Ill change it right away. This was actually giving me problems.